pages/en/contrib/Reporting-issues.md

```diff
@@ -85,18 +98,48 @@ When reporting bugs, please include at least all the following:
 
 Consider using [NodeReport](https://developer.ibm.com/node/2016/08/18/nodereport-first-failure-data-capture-for-node-js/) to gather and report the relevant information.
 
-## 2. Report in appropriate channel
+#### LoopBack 4.x bugs
+
+To report a bug in LoopBack version 4.x or newer:
+
+1. Fork [loopback-next](https://github.com/strongloop/loopback-next) monorepo.
+2. Create a new feature branch, e.g. `git checkout -b my-issue`
+3. Pick one of the projects in
+  [`examples`](https://github.com/strongloop/loopback-next/tree/master/examples)
+  directory as a starting point for your application reproducing the problem. The
+  [Todo](https://github.com/strongloop/loopback-next/tree/master/examples/todo)
+  example should work well for most users.
+4. Modify the selected example project to reproduce the issue you are
+   experiencing. Add the required code and configuration. When adding new
+   dependencies, don't forget to save them to `package.json` too.
+5. If possible, encode steps to reproduce in an automated test we can run
+   via `npm test`.
+6. Commit the changes (`git commit`) and push the feature branch to GitHub
+   (`git push -u origin my-branch-name`)
+6. Create an issue in the appropriate GitHub repository with steps to reproduce
```

Comment:
Should we mention that they create it in either `loopback` or `loopback-next` if they don't know the appropriate repository?

Reply:
Ah good point. I don't think it's worth repeating if that's the case. Adding a link to `Report in appropriate channel` sounds like a good middle ground.

Reply:
a dup 6. here.
Should be 7.